assets/popup.css

```diff
@@ -72,13 +72,21 @@
   }
 }
 
-body {
-  margin: 0;
-  padding: 0;
+html[data-theme] {
```

Comment:
This applies to Connect only, while the one below applies to BE only.

Not the nicest, but we can reorganize these shared files after the monorepo is setup.